ssfevents.caep.sgnl.yaml

```diff
@@ -0,0 +1,417 @@
+deliveryMethod: "eventPush"
+
+entities:
+  SessionRevoked:
+    # https://openid.net/specs/openid-caep-1_0-03.html#name-session-revoked.
+    displayName: Session Revoked
+    externalId: https://schemas.openid.net/secevent/caep/event-type/session-revoked
+    description: Session Revoked Event
+    pageSize: 1000
+    pagesOrderedById: false
+    attributes:
+      - name: id
+        externalId: $.jti
+        type: String
+        indexed: true
+        uniqueId: true
+      - name: subject
+        externalId: $.sub_id.id
+        type: String
+        indexed: true
```

Comment:
@marc-sgnl You OK with the default `externalId` here? The `name: subject` is mandatory and enforced by the Config API at the moment.